### PR TITLE
[stable/kubernetes-dashboard] add extraEnv

### PR DESCRIPTION
**Pull request by bskim45**
#### What this PR does / why we need it:

Added option to set any number of extra ENV variables.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/kubernetes-dashboard/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: kubernetes-dashboard
-version: 1.5.2
+version: 1.5.3
```
Comment (cpanato) [inline stable/kubernetes-dashboard/Chart.yaml]: should be
```suggestion
version: 1.6.0
```
not a bug fix

**Comment by bskim45** [comment]
bumped to `1.6.0`

**Comment by bskim45** [comment]
Seems like the other PR merged one step faster than this. I'll rebump the version.

**Comment by bskim45** [comment]
guys I just resolved all conflicts. Can you `lgtm` this?

**Comment by desaintmartin** [comment]
Thanks!
/lgtm

**Pull request merged**